<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPrisoner</class>
 <widget class="QDialog" name="AddPrisoner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font-family: century gothic;

}

AddPrisoner{
	background-color:#001F3F
}

#label,#label_2,#label_3,#label_4,#label_6,#label_5{
	font-size:16px;
	color:white;
}

#submit_btn{
	background-color : #cb0000 ;
	border-radius : 10px;
	padding:10px;
	font-size: 15px;
	font-color:
}

#submit_btn:hover{
	background-color: #850101;
}

#label_7{
	font-size:20px;
	font-weight:bold;
	color:white;
}


</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Add Prisoner:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="nameLineEdit">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Age:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="ageLineEdit">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Crime</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="crimeLineEdit">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Crime Details</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="crimeDetailsLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Sentence </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="sentenceLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Entry Date</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDateEdit" name="entryDateEdit"/>
   </item>
   <item>
    <widget class="QPushButton" name="submit_btn">
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
